Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -30,11 +34,22 @@ @implementation WMFArticlePreviewTableViewCell
 
 #pragma mark - Setup
 
+- (void)blurAndShowLoadingIndicator {
```

Comment:
Agreed. I've been a little shy lately about making things properties if they're not just super simple value assignments, but in this case it seems ok.